storage/ndb/rest-server/rest-api-server/internal/integrationtests/batchfeaturestore/performance_test.go

```diff
@@ -0,0 +1,171 @@
+/*
+ * This file is part of the RonDB REST API Server
+ * Copyright (c) 2023 Hopsworks AB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package batchfeaturestore
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"net/http"
+	"testing"
+
+	"hopsworks.ai/rdrs/internal/config"
+	"hopsworks.ai/rdrs/internal/feature_store"
+	"hopsworks.ai/rdrs/internal/integrationtests/testclient"
+	"hopsworks.ai/rdrs/internal/testutils"
+	"hopsworks.ai/rdrs/pkg/api"
+	"hopsworks.ai/rdrs/resources/testdbs"
+
+	"hopsworks.ai/rdrs/internal/integrationtests"
+	fshelper "hopsworks.ai/rdrs/internal/integrationtests/feature_store"
+	"hopsworks.ai/rdrs/internal/log"
+)
+
+/*
+	 The number of parallel client go-routines spawned in RunParallel()
+	 can be influenced by setting runtime.GOMAXPROCS(). It defaults to the
+	 number of CPUs.
+
+	 This test can be run as follows:
+
+		 go test \
+			 -test.bench BenchmarkSimple \
+			 -test.run=thisexpressionwontmatchanytest \
+			 -cpu 1,2,4,8 \
+			 -benchmem \
+			 -benchtime=100x \ 		// 100 times
+			 -benchtime=10s \ 		// 10 sec
+			 ./internal/integrationtests/pkread/
+*/
+
+const totalNumRequest = 100000
+
+func Benchmark(b *testing.B) {
+	for _, n := range []int{10, 50, 100} {
+		run(b, "fsdb001", "sample_1", 1, n)
+	}
+}
+
+func Benchmark_join(b *testing.B) {
+	for _, n := range []int{5, 25, 50} {
+		run(b, "fsdb001", "sample_1n2", 1, n)
+	}
+}
+
+func getSampleData(n int, fsName string, fvName string, fvVersion int) ([][]interface{}, []string, []string, error) {
+	switch fmt.Sprintf("%s|%s|%d", fsName, fvName, fvVersion) {
+	case "fsdb001|sample_1|1":
+		return fshelper.GetNSampleData("fsdb001", "sample_1_1", n)
```

Comment:
there are constraints defined in testdb package for database names. use those constants instead of hard coding the names here. 
you can also add constants for tables names